src/components/chip-set/chip-set.scss

```diff
@@ -272,12 +273,16 @@ limel-icon.mdc-chip__icon.mdc-chip__icon--leading {
 .has-leading-icon {
     &:not(.has-chips) {
         .mdc-text-field__input {
-            padding-left: functions.pxToRem(40);
+            padding-left: $leading-icon-space;
         }
 
         .mdc-floating-label {
-            &:not(.lime-floating-label--float-above) {
-                padding-left: functions.pxToRem(40);
+            padding-left: $leading-icon-space;
+            max-width: calc(
+                100% - #{$leading-icon-space}
+            ); // minus space taken by the leading-icon
```

Comment:
This feels like a very superfluous comment, don't you think? The code literally says "one hundred percent minus leading icon space"… 😄

Reply:
😄 I think I wrote it before I made the variable for it > then thought why not make a variable instead of commenting > then I forgot to remove the comment 

hahah

Reply:
😄 👌